pypi_count/py_counter.py

```diff
@@ -83,3 +90,18 @@ def count_classes_with_docstring(self):  # pylint: disable=R1710
                 total.append(count)
                 count += 1
         return count
+
+    def count_function_parameters(self):
+        """Counting the parameters within a function."""
+        functions = m.findall(self.search, m.FunctionDef())
+        param_result = 0
+        for func in functions:
+            if func.name.value == self:
+                param_result = len(func.params.params)
+        return param_result
+
+
+    def count_assignment_statements(self):
+        """Counting the assignment statement."""
+        assignment_statement = m.findall(self.search, m.Assign())
```

Comment:
This should also count `m.AugAssign()`s.

Reply:
@guzmanr04